pypy/tool/release/repackage.sh

```diff
@@ -1,11 +1,11 @@
 #! /bin/bash
 
 # Edit these appropriately before running this script
-pmaj=3  # python main version: 2 or 3
-pmin=9  # python minor version
+pmaj=2  # python main version: 2 or 3
+pmin=7  # python minor version
```

Comment:
@mattip is this correct to merge to pypy3.9? or should the change be removed in the merge commit?

Reply:
It doesn't matter. I fix up `pmaj`, `pmin` for each repackaging run, so the values that end up committed here are kind of random